apps/barista-design-system/src/app/app.ts

```diff
@@ -0,0 +1,165 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// tslint:disable: indent object-literal-key-quotes quotemark trailing-comma max-line-length no-duplicate-imports max-file-line-count
+
+import { DOCUMENT } from '@angular/common';
+import { Component, Inject, OnInit, OnDestroy } from '@angular/core';
+import { Event, NavigationEnd, Router } from '@angular/router';
+import {
+  distinctUntilChanged,
+  filter,
+  map,
+  pluck,
+  share,
+  takeUntil,
+} from 'rxjs/operators';
+import {
+  BaPageService,
+  getPageKeyFromUrl,
+  getUrlPathName,
+} from '../shared/services/page.service';
+import {
+  BaContentTypes,
+  BaPageLayoutType,
+} from '@dynatrace/shared/barista-definitions';
+import { Meta, MetaDefinition, Title } from '@angular/platform-browser';
+import { Subject } from 'rxjs';
+
+const PAGE_THEME_MAP = new Map<string, string>([
+  ['brand', 'purple'],
+  ['resources', 'blue'],
+  ['components', 'royalblue'],
+  ['patterns', 'turquoise'],
+]);
+
+export const DEFAULT_PAGE_THEME = 'turquoise';
+
+export interface BaBreadCrumb {
+  label: string;
+  url: string;
+}
+
+@Component({
+  selector: 'ba-app',
+  templateUrl: './app.html',
+  styleUrls: ['./app.scss'],
+})
+export class BaApp implements OnInit, OnDestroy {
+  _url$ = this._router.events.pipe(
+    filter((event: Event) => event instanceof NavigationEnd),
+    pluck('url'),
+    distinctUntilChanged(),
+    share(),
+  );
+
+  /** The stream that holds the current breadcrumbs */
```

Comment:
```suggestion
  /** @internal The stream that holds the current breadcrumbs */
```

Reply:
Please @tomheller explain me why we need the internal inside an app.
In a library I totally understand but I don't get the point inside an app.

Nobody consumes it and I can make it private or leave it public.

Reply:
Because you are doing it everywhere else. 
But if you don't want to change it, consider it a NIT!